AirCasting/SingleSessionSynchronizer/SessionUpdateService.swift

```diff
@@ -5,19 +5,26 @@ import Foundation
 import Resolver
 
 protocol SessionUpdateService {
-    func updateSession(session: SessionEntity, completion: @escaping () -> Void)
+    func updateSession(session: SessionEntity, completion: @escaping (Result<UpdateSessionReturnedData, Error>) -> Void)
 }
 
-class DefaultSessionUpdateService: SessionUpdateService {
+struct UpdateSessionReturnedData: Codable {
+    let version: Int
+}
```

Comment:
Maybe we could make a new file for this as for other structs we do?

Reply:
I don't think that's necessary, we can move this struct above the SessionUpdateService protocol definition because we're using it there so it would be easier to read this file top-down